tensorrt-sys/trt-sys/TRTContext/TRTContext.h

```diff
@@ -8,20 +8,21 @@
 #define LIBTRT_TRTCONTEXT_H
 
 #include <stddef.h>
+#include "../DLLExport.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 struct Context;
-typedef struct Context Context_t;
+using Context_t = struct Context;
```

Comment:
We can't use the `using` keyword in the `.h` wrapper headers because bindgen doesn't understand the `using` keyword as a valid type when creating the Rust bindings.